sass/components/_post.scss

```diff
@@ -2239,6 +2240,12 @@
     }
 }
 
+#post_textbox::after {
+    display: inline-block;
+    clip-path: polygon(100% 0, 100% 100%, 0 100%);
+    content: '';
+}
```

Comment:
does this do anything? AFAIK `::after` cannot be used in input elements

Reply:
lol ... you are right ... this is a remnant of something I tried ... will remove it